### DISCUSSION TITLE
cmd: "lightdash dbt run -s model" create Error running postgres query: Error: The server does not support SSL connections

### DISCUSSION THREAD

**Discussion by DeoDepari** [lightdash/lightdash#6520] [Q&A] [answered]
Hi Everyone, so i wanted to add a new database, scheme and tables into my dbt project in lightdash.
I use this article to help guide me:
https://docs.lightdash.com/guides/adding-tables-to-lightdash/

but on running this cmd:
"lightdash dbt run -s model"

Here is the result and issue that i got
07:13:29  Running with dbt=1.5.3
07:13:29  Registered adapter: postgres=1.5.3
07:13:29  Found 3 models, 4 tests, 0 snapshots, 0 analyses, 307 macros, 0 operations, 0 seed files, 0 sources, 0 exposures, 0 metrics, 0 groups
07:13:29  
07:13:29  Concurrency: 10 threads (target='dev')
07:13:29
07:13:29  1 of 1 START sql table model public.dvdrental .................................. [RUN]
07:13:29  1 of 1 OK created sql table model public.dvdrental ............................. [SELECT 200 in 0.18s]
07:13:29  
07:13:29  Finished running 1 table model in 0 hours 0 minutes and 0.44 seconds (0.44s).
07:13:29
07:13:29  Completed successfully
07:13:29
07:13:29  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
Generated .yml files:
✖   Failed to generate dvdrental.yml
Error running postgres query: Error: The server does not support SSL connections

**Comment by durantsmedia** [comment]
It looks like you might need to adjust you connection settings to your database to  sslmode: disable

**Reply by DeoDepari** [reply]
I have tried disabling sslmode in Lightdash
![image](https://github.com/lightdash/lightdash/assets/45847038/f8c747ed-5f6e-428d-8e91-9fe84ef69f72)

I also have change all sslmode / ssl into "disable"
![image](https://github.com/lightdash/lightdash/assets/45847038/7b819b75-f3e6-4d99-b7b6-d562d11f0ddb)

In Postgre if i use “show ssl” here is the result:
![image](https://github.com/lightdash/lightdash/assets/45847038/97714a4b-2c48-458d-8cdf-e0115da39480)

Running this cmd also doesnt work:
Command Attempt:
lightdash dbt run --ssl=disable
lightdash dbt run -ssl=disable dvdrental
lightdash dbt run -s -ssl=disable dvdrental
lightdash dbt run -s -sslmode=prefer dvdrental.sql

unfortunately still doesn't work, are there any config/options that i miss?

**Reply by durantsmedia** [reply, marked as answer]
Update your profile.yml to something like this.

    internal:
      type: postgres
      threads: 4
      host: 192.168.3.210
      port: 5432
      user: xxxxx
      pass: xxxxx
      dbname: xxxxx
      schema: public
      sslmode: disable

**Reply by DeoDepari** [reply]
Thank you so much, adding sslmode:disable in profile.yml works
